docs/tutorials/notebooks/tracking_progress.ipynb

```diff
@@ -7,21 +7,15 @@
     "tags": []
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    print(f"The last Sinkhorn iteration has converged: {has_converged}")
</pre><p>I don't think it's really necessary to print the convergence of the last Sinkhorn.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>